product_docs/docs/pgd/5.6/reference/catalogs-internal.mdx

```diff
@@ -92,6 +92,12 @@ A view of the `bdr.event_history` catalog that displays the information in a mor
 human-friendly format. Specifically, it displays the event types and subtypes
 as textual representations rather than integers.
 
+### `bdr.local_leader_change`
+
+This is a local cache of recent portion of leader change history. It has the same fields as [`bdr.leader`](/pgd/5.6/reference/catalogs-visible#bdrleader), except that it is an ordered set of (node_group_id, leader_kind, generation) instead of a map tracking merely the current version.
```

Comment:
```suggestion 
This is a local cache of the recent portion of leader change history. It has the same fields as [`bdr.leader`](/pgd/5.6/reference/catalogs-visible#bdrleader), except that it is an ordered set of (node_group_id, leader_kind, generation) instead of a map tracking merely the current version.